lux/vislib/matplotlib/ScatterChart.py

```diff
@@ -82,10 +84,29 @@ def initialize_chart(self):
                 plot_code += f"cbar = plt.colorbar(sm, label='{color_attr_name}')\n"
                 plot_code += f"cbar.outline.set_linewidth(0)\n"
             else:
-                scatter = self.ax.scatter(x_pts, y_pts, c=vals, cmap="Set1")
-                plot_code += f"scatter = ax.scatter(x_pts, y_pts, c={vals}, cmap='Set1')\n"
+                if len(unique) >= 16:
+                    unique = unique[:16]
+
+                maxlen = 0
+                for i in range(len(unique)):
+                    unique[i] = str(unique[i])
+                    if len(unique[i]) > 26:
+                        unique[i] = unique[i][:26] + "..."
```

Comment:
This is not showing up.

Reply:
what do you mean by this?

Reply:
I meant the abbreviation "..." is not showing up in the legend, see comment above.

Reply:
o hmm i think it was working on my end
![109049161-c787d900-768c-11eb-95d0-44363d65fabc](https://user-images.githubusercontent.com/11529801/109407450-c75e3680-7935-11eb-8460-b11dd5b7a354.png)

Reply:
are you seeing something differently

Reply:
I'm able to see it.

Reply:
![image](https://user-images.githubusercontent.com/5554675/109598581-7c6e3a00-7b54-11eb-9a83-7b7e7f735213.png)

Oops sorry, I meant when the `...` vertically (not horizontally) when there are too many color categories.

>  In the Brand legend example, there are only 16 colors displayed in the legend but there is 29 different Brand in the dataset.

Reply:
Oh this is a good point, a user might be confused why a specific category is not showing up, so something in the legend would be nice to address this. Another idea rather than using the '...' (which seems a bit off to me) could be to use multiple columns. Try maybe something like this: https://stackoverflow.com/questions/58344791/how-to-handle-categorical-legends-with-too-many-entries-to-match-a-fixed-chart-h

Reply:
ohh the "..." is not for the length of the legend, but rather the individual categories. I checked and Altair also didn't have a "..." at the end of the list, so didn't include it in this PR. If you want, maybe I can tackle that issue separately

Reply:
Added an issue https://github.com/lux-org/lux/issues/295 for this new change